archunit/src/main/java/com/tngtech/archunit/core/domain/JavaMethod.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014-2022 TNG Technology Consulting GmbH
+ * Copyright 2014-2023 TNG Technology Consulting GmbH
```

Comment:
I guess we should isolate this _"Happy new year, code base!"_ change into a separate commit (that also fixes all java files, which happens automatically when building the project).

Reply:
Yeah for some reason my IntelliJ or some other plugin automatically made this change for all the classes with this documentation, I forgot to undo the changes on this file itself but did them for all other classes. 😅